OpenConsole.sln

```diff
@@ -466,7 +466,6 @@ Global
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|ARM64.ActiveCfg = Release|ARM64
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|ARM64.Build.0 = Release|ARM64
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|x64.ActiveCfg = Release|x64
-		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|x64.Build.0 = Release|x64

```

Comment:
Should we remove the rest of the AuditMode stuff from these projects, or just leave them with only the `AuditMode|x64.Build.0` removed?

Reply:
They don't actually have an audit mode configuration defined in the projects. It's just that the solution has one and it has to map to SOMETHING in the individual projects.

Reply:
OK. we talked and you meant the x86 and ARM ones too. I dropped those too.